media/css/firefox/family/components/_hero.scss

```diff
@@ -20,71 +20,29 @@
         }
```

Comment:
🤦‍♀️ my fault for coding desktop first, but the padding on L16 should start around `$spacing-lg` size to match figma design
<img width="195" alt="Screen Shot 2022-08-30 at 2 43 42 PM" src="https://user-images.githubusercontent.com/19650432/187517955-71e1f28f-0ab7-4982-b19f-7e973ae307dc.png">

probably around medium or large media query we can increase to `$spacing-2xl`